Week2.py

```diff
@@ -0,0 +1,90 @@
+
+import unittest
+
+def find_triplets(string, target_sum):
```

Comment:
I like the initiative with including target_sum.  While not required by the question, it is an obvious extension of the functionality that could be asked for a follow up.  Also in the real world, it often happens that a more generalized function is needed for maximum reuse

Reply:
I would changethis, however, so that the input is not a string but the actual array.  You should move the string splitting functionality to some other method and let this function focus only on finding the triplets.  Other users of your function might not use the same string format, so having them input an array would make it more reusable